static/js/MapView.js

```diff
@@ -64,6 +64,15 @@ MapView = (function($, L, Models, Config) {
         });
         self.tabsControl.addTo(self.map);
 
+        self.sliderControl = L.control.sliderControl({
+            position: "bottomright",
```

Comment:
Yeah I agree. It's in the bottom corner right now because that's where I
demoed it. :)
On Jan 12, 2015 11:47 PM, "Kelsey Jordahl" <notifications@github.com> wrote:

> In static/js/MapView.js
> <https://github.com/enthought/tabs/pull/57#discussion-diff-22844365>:
>
> > @@ -64,6 +64,15 @@ MapView = (function($, L, Models, Config) {
> >          });
> >          self.tabsControl.addTo(self.map);
> >
> > +        self.sliderControl = L.control.sliderControl({
> > +            position: "bottomright",
>
> It doesn't seem to get in the way of the other control if set to topright.
> Probably it could be laid out better as part of the TABSControl, though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/enthought/tabs/pull/57/files#r22844365>.
>